docs/metrics-events.md

```diff
@@ -172,7 +188,7 @@ with the following fields:
 |`timestamp`|The time at which the event occurred.|
 |`type`|The name of the event.|
 |`uid`|The user id. An opaque token, HMACed to avoid correlation back to FxA user db.|
-|`device_id`|Optional. The id of the device record.  This *does* correlate back to a record the FxA user db.|
+|`device_id`|Optional. The id of the device record, HMACed to avoid correlation back to the FxA user db.|
```

Comment:
Oh, my bad, I just went looking for a fix and there is none. I think I just assumed the magic bug fairy was fixing that one, sorry. I'll reinstate the previous text and open a `puppet-config` PR for the change separately.